lib/spandex_datadog/api_server.ex

```diff
@@ -133,8 +150,9 @@ defmodule SpandexDatadog.ApiServer do
   end
 
   @spec send_and_log([Trace.t()], State.t()) :: :ok
-  def send_and_log(traces, %{verbose?: verbose?} = state) do
+  def send_and_log(traces, %{container_id: container_id, verbose?: verbose?} = state) do
     headers = @headers ++ [{"X-Datadog-Trace-Count", length(traces)}]
+    headers = headers ++ List.wrap(if container_id, do: {"Datadog-Container-ID", container_id})
```

Comment:
reference to this header in DataDog's Node.js client
https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/exporters/agent/request.js#L26